src/multibody/model.hxx

```diff
@@ -31,7 +31,7 @@ namespace se3
   inline std::ostream& operator<< (std::ostream & os, const Model & model)
   {
     os << "Nb bodies = " << model.nbody << " (nq="<< model.nq<<",nv="<<model.nv<<")" << std::endl;
-    for(Model::Index i=0;i<(Model::Index)(model.nbody);++i)
```

Comment:
I guess I should change `"Nb bodies  = " << model.nbody` by  `"Nb joints  = " << model.njoint` ?